src/handlers/healthcheck.spec.ts

```diff
@@ -1,46 +0,0 @@
-import { APIGatewayProxyEvent, Context, Callback, APIGatewayProxyResult } from 'aws-lambda';
```

Comment:
Maybe leave this test since we're keeping the endpoint. You'll just want to remove the redis behaviors.